sdk/core/core-http/src/policies/msRestUserAgentPolicy.native.ts

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+/*
+ * NOTE: When moving this file, please update "browser" section in package.json
+ * and "plugins" section in webpack.testconfig.ts.
```

Comment:
do we still have `webpack.testconfig.ts` ?

Reply:
ah, forgot to remove this when pasting.  I will remove it from browser version as well.